src/pages/Popup/index.html

```diff
@@ -1,11 +1,14 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <title></title>
-  </head>
 
-  <body class="chrome-extension-react">
-    <div id="extension-app-container"></div>
-  </body>
-</html>
+<head>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap" rel="stylesheet">
+  <meta charset="UTF-8" />
+  <title></title>
+</head>
+
+<body class="chrome-extension-react">
+  <div id="extension-app-container"></div>
```

Comment:
Don't make any changes to any HTML/CSS files
Make any necessary changes in the JSX files itself
All the changes are bundled and added to the html files